opentelemetry-zipkin/src/exporter/mod.rs

```diff
@@ -141,7 +141,9 @@ impl ZipkinPipelineBuilder {
 
     /// Install the Zipkin trace exporter pipeline with a simple span processor.
     #[allow(deprecated)]
-    pub fn install_simple(mut self) -> Result<Tracer, TraceError> {
+    pub fn install_simple(
```

Comment:
not related to this PR - but `ZipkinPipelineBuilder` should be replaced with `ZipkinBuilder` and made consistent with the OTLP as done in #2221. The crate be only responsible to provide builder to create Zipkin exporter, and no convenience wrappers to create TracerProvider / Processors.